core/forms.py

```diff
@@ -105,6 +117,80 @@ def __init__(self, *args, **kwargs):
             visible.field.widget.attrs["class"] = "form-button-style py-2"
 
 
+class HackerSettingForm(forms.ModelForm):
+    country = forms.CharField(max_length=255)
+    city = forms.CharField(max_length=255)
+    school = forms.CharField(max_length=255, required=False)
+    education = forms.ChoiceField(
+        choices=HACKATHON_EDUCATION_CHOICES, widget=forms.Select
+    )
+    username = forms.CharField(max_length=255)
+    email = forms.CharField(max_length=255)
+    first_name = forms.CharField(max_length=255)
+    last_name = forms.CharField(max_length=255)
+    birthday = forms.DateField(widget=forms.TextInput(attrs={"type": "date"}))
+    personal_website = forms.CharField(max_length=255, required=False)
+
+    class Meta:
+        model = Hacker
+        exclude = [
+            "objects",
+            "saved",
+            "saved_categories",
+            "notification_policy",
```

Comment:
They should be able to config notif policy but potentially out of scope for this commit